examples/curations.yml

```diff
@@ -48,3 +48,8 @@
       but it is not part of the distributed .tar.gz package, see the `README.md` which says: \
       Ramda logo artwork © 2014 J. C. Phillipps. Licensed Creative Commons CC BY-NC-SA 3.0"
     concluded_license: "MIT"
+
+- id: "Maven:ort.junit:junit-bom"
```

Comment:
How about using a real-life example like https://repo1.maven.org/maven2/org/jetbrains/kotlin/kotlin-bom/1.4.0/kotlin-bom-1.4.0.pom? Or if using a synthetic example, better come up with completely fake names rather than mixing fake names and real names like "ort" and "junit".

Reply:
Speaking of real-life examples, I guess the missing bit here still is to automatically set `isMetaDataOnly` to `true` e.g. for Maven packages whose `packaging` is `pom`, right?

Reply:
> How about using a real-life example like https://repo1.maven.org/maven2/org/jetbrains/kotlin/kotlin-bom/1.4.0/kotlin-bom-1.4.0.pom? Or if using a synthetic example, better come up with completely fake names rather than mixing fake names and real names like "ort" and "junit".

It was supposed to be a real life example, e.g. it was a typo **ort -> org**. Done.

Reply:
> Speaking of real-life examples, I guess the missing bit here still is to automatically set `isMetaDataOnly` to `true` e.g. for Maven packages whose `packaging` is `pom`, right?

I'm going to implement this in a separate PR, see also the description of this PR.